### PR TITLE
DOMA-2113 fix element layout

### PR DESCRIPTION
**Pull request by VKislov**
Before: 
image

After: 
image

### CODE REVIEW

File: apps/condo/domains/ticket/components/TicketForm/CreateTicketForm.tsx [hunk 1]
```diff
@@ -32,7 +34,36 @@ export const CreateTicketActionBar = ({ handleSave, isLoading }) => {
                     const { property, details, placeClassifier, categoryClassifier, deadline } = getFieldsValue(REQUIRED_TICKET_FIELDS)
                     const disabledCondition = !property || !details || !placeClassifier || !categoryClassifier || !deadline
 
-                    return (
+                    return isSmall ? (
+                        <ActionBar isFormActionBar>
+                            <Col>
+                                <Space direction='vertical' size={24} >
```
Comment (pahaz) [inline apps/condo/domains/ticket/components/TicketForm/CreateTicketForm.tsx:40]: Only Row can me placed inside the Col element: https://ant.design/components/grid/

File: apps/condo/domains/ticket/components/TicketForm/CreateTicketForm.tsx [hunk 2]
```diff
@@ -8,13 +8,14 @@ import { Ticket } from '@condo/domains/ticket/utils/clientSchema'
...
 import { useOrganization } from '@core/next/organization'
-import { Form, Space } from 'antd'
+import { Col, Form, Row, Space } from 'antd'
 import { useRouter } from 'next/router'
 import React, { useCallback, useMemo } from 'react'
 import { useApolloClient } from '@core/next/apollo'
 import { get } from 'lodash'
 import dayjs from 'dayjs'
 import isToday from 'dayjs/plugin/isToday'
+import { useLayoutContext } from '../../../common/components/LayoutContext'
```
Comment (sitozzz) [inline apps/condo/domains/ticket/components/TicketForm/CreateTicketForm.tsx:18]: Absolute import plz
```suggestion
import { useLayoutContext } from '@condo/domains/common/components/LayoutContext'
```

**Review by akarjakin — APPROVED** [review]

File: apps/condo/domains/ticket/components/TicketForm/CreateTicketForm.tsx [hunk 3]
```diff
@@ -34,26 +36,29 @@ export const CreateTicketActionBar = ({ handleSave, isLoading }) => {
...
-                                    isVisible={disabledCondition}
-                                    property={property}
-                                    details={details}
-                                    placeClassifier={placeClassifier}
-                                    categoryClassifier={categoryClassifier}
-                                    deadline={deadline}
-                                />
-                            </Space>
+                            <Col>
+                                <Row gutter={[0, 24]}>
```
Comment (sitozzz) [inline apps/condo/domains/ticket/components/TicketForm/CreateTicketForm.tsx:40]: move array to local component constants

File: apps/condo/domains/ticket/components/TicketForm/CreateTicketForm.tsx [hunk 4]
```diff
@@ -34,26 +36,29 @@ export const CreateTicketActionBar = ({ handleSave, isLoading }) => {
...
+                                    <Button
+                                        key='submit'
+                                        onClick={handleSave}
+                                        type='sberDefaultGradient'
+                                        loading={isLoading}
+                                        disabled={disabledCondition}
+                                        data-cy={'ticket__submit-button'}
+                                        style={{ marginRight: '12px' }}
```
Comment (sitozzz) [inline apps/condo/domains/ticket/components/TicketForm/CreateTicketForm.tsx:48]: move style object to constants

**Pull request merged**